homeassistant/components/homewizard/config_flow.py

```diff
@@ -28,6 +28,21 @@ def __init__(self) -> None:
         """Initialize the HomeWizard config flow."""
         self.config: dict[str, str | int] = {}
 
+    async def async_step_import(self, import_config: dict) -> FlowResult:
+        """Handle a flow initiated by older `homewizard_energy` component."""
+        _LOGGER.debug("config_flow async_step_import")
+
+        self.hass.components.persistent_notification.async_create(
```

Comment:
Should we make this a persistent notification or a log?

We general log these

Reply:
Also fine! Just took over the same behaviour from this migration: https://github.com/home-assistant/core/pull/59698

Reply:
I think a persistent notification is better, easy to miss a log.